packages/copyable/src/Copyable/Copyable.spec.tsx

```diff
@@ -113,14 +113,15 @@ describe('packages/copyable', () => {
             },
           );
 
-          await waitFor(() => expect(getByText('Copied!')).toBeVisible());
+          const tooltip = getByText('Copied!');
 
           // Tooltip should remain visible for a while
+          await waitFor(() => expect(tooltip).toBeVisible());
+          
           await new Promise(resolve => setTimeout(resolve, 1000));
-          expect(getByText('Copied!')).toBeVisible();
 
           // Tooltip should eventually disappear
-          await waitForElementToBeRemoved(() => queryByText('Copied!'));
+          await waitFor(() => expect(tooltip).not.toBeVisible());
```

Comment:
Curious, do we know what was causing the flakiness? I would feel like switching
```ts
await new Promise(resolve => setTimeout(resolve, 1000));
```
to
```ts
await waitFor(() => expect(getByText('Copied!')).toBeVisible());
```
alone would have fixed the test since it was failing on line 120 and not on line 123. I'm wondering why 123 needs a change at all?

Also curious, why are we waiting for 1 second on line 119/121?

Reply:
I wonder if it would be better to just lengthen the `waitFor` timeout. If it's just an early timeout, I believe that should solve it without the need for pausing execution entirely.
```ts
await waitFor(
  () => expect(tooltip).not.toBeVisible(), 
  { timeout: TOOLTIP_VISIBLE_DURATION } // or more if necessary?
);
```